templates/base.html

```diff
@@ -56,22 +56,87 @@ <h1 class="brand m-0 h4"><a href="/">2021 Georgia Early Vote Ballot Tracker</a><
+            <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click the button to run the search, we won't post anything to your wall or spam your friends.</em></p>
+            <!-- TODO: Add something about needing to be logged into FB, otherwise you get an error -->
+            <!-- TODO: Add something about what the search is, if they are hesitant to click the button -->
+            <!-- TODO: Deep link to FB on mobile? -->
+            <hr class='my-4'>
+          </div>
+
+          <div class='mt-4'>
+            <h4>2️⃣ Let them know there's an election</h4>
+            <p>In most elections, over 30% of voters don't know when the election is. Make sure your friends know that the election is on <strong>Tuesday, January 5, 2021</strong>. Anyone can vote early starting on <strong>Monday, December 14, 2020</strong>.</p>
+            <!-- TODO: Add image of copy for text and email, make "copyable" -->
+
+            <!-- TODO: See if this works, I have an Android rn -->
+            <a href="sms://body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Text a friend</a>
+            
+            <a href="mailto:?to=&subject=Vote%20early%20in%20the%20Georgia%20runoff%20election&body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Email a friend</a>
+            
+            <hr class='my-4'>
+          </div>
+
+          <div class='mt-4'>
+            <h4>3️⃣ Follow up!</h4>
+            <p>Research shows that people vote when you ask them to. Make sure to follow up and ask your friends if they've voted!</p>
+
+            {% block content %}{% endblock %}
+          </div>
+
+          <div class='mt-4'>
+            <h4>4️⃣ Support our friends on the ground</h4>
+            <p>Once you've talked to your friends, volunteer with one of these amazing organizations working in Georgia to make sure every voice is heard and every vote is counted.</p>
+            <!-- TODO: Insert logos here -->
+          </div>
+
+          ///<br><br>
+
+          <!-- For FAQ? -->
```

Comment:
i don't see any of the data in question here. agree we should get rid of this disclaimer except on pages where we are displaying private data, such as on a search results page.

Reply:
maybe this becomes something like:
```
      {% block disclaimer %}
      {% endblock %}
```
and then the search results page could override that?

Reply:
i moved this to search results, if you want it elsewhere lmk?

Reply:
kinda a dumb question, but i'm not sure how to DRY in jinja(?) re: https://github.com/vote/gaballot/pull/2/commits/81f36357d4ba1daf2fbc5013ea8058d68abc7e88

Reply:
Move to https://github.com/vote/gaballot/issues/3 for now, which I can do